lib/app/app.dart

```diff
@@ -16,6 +16,7 @@ import 'package:receipe_app/ui/views/login/login_view.dart';
     AdaptiveRoute(page: StartupView, initial: true),
     AdaptiveRoute(page: OnboardingView),
     AdaptiveRoute(page: LoginView),
+    MaterialRoute(page: HomepageView),
```

Comment:
Change `MaterialRoute` to `AdaptiveRoute`

Reply:
close